### DISCUSSION TITLE
How to turn off data type mismatch warning - unsigned integer not supported

### DISCUSSION THREAD

**Discussion by tg0h** [sequelize/sequelize#14557] [Help & Questions] [answered]
# How to turn off datatype mismatch warning

How do I turn off the warning `(sequelize) Warning: SQLite does not support 'INTEGER' with UNSIGNED or ZEROFILL. Plain 'INTEGER' will be used instead.`

## Background
I defined an unsigned integer in a column. I am aware that the unsigned integer is not supported in SQLite. (https://sequelize.org/docs/v7/other-topics/other-data-types/#integers) 

However, I use a MySQL db in production but an SQLite db for testing. While running my tests, I am getting the warning above which is obscuring my test output.

I have tried setting the logging option to false (https://sequelize.org/docs/v6/getting-started/#logging) but it did not work. Is there any way to turn off the warning?

**Comment by ephys** [comment, marked as answer]
There is no option to disable warnings, they are not subject to the logging option. Ideally the logging option would let you choose which verbosity you want (debug, info, warn, error).

You should consider running your tests on the same type of database as production instead, MySQL and SQLite are very different
